RecoLocalCalo/EcalRecAlgos/BuildFile.xml

```diff
@@ -11,6 +11,9 @@
 <use   name="root"/>
 <use   name="rootminuit"/>
 <use   name="eigen"/>
+<use   name="cuda"/>
+<use   name="DataFormats/EcaRecHitSoA"/>
+<flags CUDA_FLAGS="-Xptxas -v"/>
```

Comment:
Please don't do this in production.

Rather, leave the `CUDA_FLAGS` out of the BuildFile.xml, and use them when you build with scram, e.g.
```
USER_CUDA_FLAGS="-Xptxas -v" scram b
```

Reply:
removed in #335